services/QuillLMS/db/migrate/20220830205005_create_activity_pack_sequence_units.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class CreateActivityPackSequenceUnits < ActiveRecord::Migration[6.1]
+  def change
+    create_table :activity_pack_sequence_units do |t|
```

Comment:
Some naming considerations here. If we rename `units` to `activity_packs` (which we should eventually), this will be `activity_pack_sequence_activity_packs`. It still may be the best name, just noting it.

Reply:
I debated doing this but I found the name really ugly and long.  However, on the second thought it does get us moving in the right direction of more clarity in our codebase so I will rename it.